src/interface/file.rs

```diff
@@ -8,7 +8,7 @@ use std::collections::HashSet;
 use std::fs::{self, File, OpenOptions};
 use std::env;
 use std::slice;
-use std::path::{PathBuf, Path};
+pub use std::path::{PathBuf, Path, Component};
```

Comment:
I know were at odds here but I do actually like these named as - "RustPathstr" "RushPathString" and "RustPathComponent"